src/main/java/org/apache/commons/imaging/palette/MedianCutQuantizer.java

```diff
@@ -130,7 +130,7 @@ public Palette process(final BufferedImage image, final int maxColors,
 
             colorGroup.paletteIndex = i;
 
-            if (colorGroup.colorCounts.size() < 1) {
+            if (colorGroup.getColorCounts().isEmpty()) {
```

Comment:
There are several other places with similar code smell, will raise a PR in the next hours to use `isEmpty()` in the rest of the code when possible.